covid-mapper/babel.config.js

```diff
@@ -1,6 +1,18 @@
+const pak = require('./package.json');
+
 module.exports = function(api) {
   api.cache(true);
   return {
     presets: ['babel-preset-expo'],
+    plugins: [
+      [
+        "module-resolver",
+        {
+          extensions: [".tsx", ".ts", ".js", ".json"],
+        },
+      ],
+      'react-native-reanimated/plugin',
+    ],
```

Comment:
If you add any more plug-ins, make sure this one stays at the bottom! I remember reading it when we installed it.